pkgdb2/templates/package.html

```diff
@@ -68,15 +68,15 @@
         <td colspan="2" align="center">
           <div id="monitoring_block">
             <input type="radio" id="no_monitoring" name="monitoring" value="0" />
-                <label for="no_monitoring" {%-
+                <label for="no_monitoring" title="Will toggle the monitor flag to monitor package for new releases" {%-
                     if package.monitoring_status == False %} class="active"{% endif
                     -%}>No monitoring</label> <br />
             <input type="radio" id="monitoring" name="monitoring" value="1" />
-                <label for="monitoring" {%-
+                <label for="monitoring" title="Will toggle the monitor flag to monitor bugs and the buildsystem for new builds" {%-
```

Comment:
Here maybe we could have something like: `Turns on monitoring and attempt to build new releases for this package`